public/js/fxa-analytics.js

```diff
@@ -124,6 +124,15 @@ function getUTMNames() {
       },
     });
 
+    // Temporarily send "View" and "Engage" pings for recommendation CTAs.
+    document.querySelectorAll(".recommendation-cta").forEach(cta => {
+      const eventLabel = cta.dataset.eventLabel;
+      ga("send", "event", "Breach Detail: Recommendation CTA", "View", eventLabel);
```

Comment:
Non-blocking fix-up: This fired even though the element wasn't showing yet. I.e., it was behind "See additional recommendations"